gstd/Cargo.toml

```diff
@@ -21,6 +21,10 @@ futures = { workspace = true, features = ["alloc"] }
 static_assertions.workspace = true
 
 [features]
-default = ["panic-handler"]
+default = ["default-stable", "nightly"]
+default-stable = ["panic-handler"]
```

Comment:
I don't think we need such precise customization so to make it as easytouse as it could be, I'd remove default-stable and separate features of oom handling and panic info in order to keep only:
- panic-handler (includes panic handler)
- nightly (includes panic info and oom handler in case of panic-handler used)

And add docs for each feature of gstd pls

Reply:
Docs added by 7048dd8.